src/pages/export.js

```diff
@@ -0,0 +1,12 @@
+import ContentCatalogue from '../components/pages/exportcontent/contentcatalogue';
+import React from 'react';
+
+const Export = () => {
+    return (
+        <div>
+            <ContentCatalogue />
+        </div>
+    );
+};
```

Comment:
Can we just return the content catalogue?

const Export = () => <ContentCatalogue />;